tests/components/kitchen_sink/test_lawn_mower.py

```diff
@@ -67,9 +67,9 @@ async def test_states(hass: HomeAssistant, snapshot: SnapshotAssertion) -> None:
             LawnMowerActivity.MOWING,
         ),
         (
-            "lawn_mower.mower_can_dock",
+            "lawn_mower.mower_can_return",
```

Comment:
Should add `lawn_mower.mower_can_return`. Not replace `lawn_mower.mower_can_dock` as it's still there